O/onnxruntime/build_tarballs.jl

```diff
@@ -0,0 +1,83 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "onnxruntime"
```

Comment:
Regarding the name, what about `ONNXRuntime`, which matches the case used in [the README](https://github.com/microsoft/onnxruntime/blob/0f5d0a091a37db48593905c33a834597338288e8/README.md)?  Or at least `ONNXRunTime` which matches the cases of [`ONNXRunTime.jl`](https://github.com/jw3126/ONNXRunTime.jl) (which arguably should have been `ONNXRuntime.jl`).  The latter might be a better choice to avoid mixing cases in the middle of the names of two related packages.

In any case remember that the directory name must match _exactly_ the name you use for this variable.

Reply:
I have renamed it to `ONNXRuntime`.